README.md

```diff
@@ -180,3 +180,49 @@ Each script saves to the same file: `rrap_canonical_[date of creation].gpkg`
 - `7_add_cruise_transit_lanes.jl` : Adds the corresponding Cruise Ship Transit Lane label to each reef where applicable.
 - `8_add_Indigenous_Protected_Areas.jl` : Adds the corresponding Indigenous Protected Areas to reefs where applicable.
 - `9_add_Indigenous_Land_Use_Agreements.jl` : Adds the corresponding Indigenous Land Use Agreement area labels to each reef where applicable.
+- `10_extract_reef_depths.jl` : Use reef features to estimate reef depths from satellite-derived raster data
+
+## Notes on feature attributes
+
+### UNIQUE IDs
+
+There needs to be a unique identifier so that all models can link their conceptualizations
+of a "reef" back to a common dataset. In Relational Databases this is known as a
+"Primary Key". Reef names should not be relied on as there are many "reefs" with
+(near-)identical or duplicate names. For example, 57.2% of reefs share the name "U/N Reef".
+
+Unfortunately, due to slight changes over time including updated processing workflows,
+revisions by external orgs, and/or changes to definitions of what a "reef" is, the assigned
+`UNIQUE_ID` no longer match.
+
+To allow cross-comparison and validation, the following IDs are included:
+
+- `UNIQUE_ID` : From the most recent GBRMPA reef feature dataset
+- `RME_UNIQUE_ID` : Taken from the most recent ReefMod Engine reef list file
+- `GBRMPA_ID` : An alternate reef id list maintained by GBRMPA (e.g., "10-330", ""10-318a")
+- `RME_GBRMPA_ID` : As above, but as defined by the ReefMod Engine reef list file
+- `LTMP_ID` : A seven character ID largely following the `GBRMPA_ID` format maintained by AIMS for the Long-Term Monitoring Program
+- `LON` and `LAT` : Pre-determined longitude and latitudes from GBRMPA reef feature set
+- `reef_name`
+
+Additionally, **all attribute names are standardized to follow `snake_case` formatting**, except
+in cases where:
+
+- Pre-existing naming convention (e.g., `UNIQUE_ID`)
+- An established acronym exists (`TUMRA`, `GBRMPA`, `LTMP`, `COTS`, `EcoRRAP`)
```

Comment:
Is Indigenous Land Use Agreement in this as well?

Reply:
Oh, those were examples...